docs/conf.py

```diff
@@ -43,17 +43,17 @@
 master_doc = 'index'
 
 # General information about the project.
-project = u'Aldryn Newsblog'
-copyright = u'2014, Tom Berger'
+project = u'Aldryn News & Blog'
+copyright = u'2014, Divio AG'
 
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
 # The short X.Y version.
-version = '0.0.1'
+version = '1.0.0'
 # The full version, including alpha/beta/rc tags.
-release = '0.0.1'
+release = '1.0.0'
```

Comment:
Is it possible to reference the version in __init__.py rather than bake it in here? Genuine question.